generated/google/protobuf/any_proto.php

```diff
@@ -101,6 +101,7 @@ public function Name(): string {
   }
 
   public function FileDescriptorProtoBytes(): string {
-    return (string)\gzuncompress(\file_get_contents(\realpath(\dirname(__FILE__)) . '/any_file_descriptor.pb.bin.gz'));
```

Comment:
Anything in the `generated/` directory is autogenerated through:
```
bazel run //:gen
```

The diffs are:
1. All the files ending in `.pb.bin.gz` were removed.
2. We inlined their content in the associated `proto.php`.